### PR TITLE
[#5465] disable realm's mixpanel analytics

### LINKED ISSUE #5465: Status Desktop feature request: Enable opting out of Mix Panel
Currently the first connection Status desktop makes is to Mixpanel:

image

I know the Mixpanel conversation is ongoing but in the meantime it would be nice if users were informed about and were able to opt out of Mixpanel analytics.

### PR DESCRIPTION
**Pull request by rasom**
fix #5465

status: ready

### CODE REVIEW

**Comment by rasom** [comment]
https://ci.status.im/job/status-react/job/desktop/job/pull-requests/job/PR-5479/1/


<blockquote><div><strong><a href="https://ci.status.im/job/status-react/job/desktop/job/pull-requests/job/PR-5479/1/">status-react » desktop » pull-requests » PR-5479 #1 [Jenkins]</a></strong></div></blockquote>

**Review by dmitryn — APPROVED** [review]

File: ci/Jenkinsfile.nightly_fastlane [hunk 1]
```diff
@@ -11,6 +11,7 @@ env.LANG="en_US.UTF-8"
 env.LANGUAGE="en_US.UTF-8"
 env.LC_ALL="en_US.UTF-8"
 env.FASTLANE_DISABLE_COLORS=1
+env.REALM_DISABLE_ANALYTICS =1
```
Comment (corpetty) [inline ci/Jenkinsfile.nightly_fastlane]: jenkins doesn't care about spaces correct?

**Review by corpetty — APPROVED** [review]
all good on this fix.  Thanks for the quick turnaround.

**Review by yenda — APPROVED** [review]

File: ci/Jenkinsfile.nightly_fastlane (hunk 1 above)
Comment (rasom) [reply ci/Jenkinsfile.nightly_fastlane]: yep, it doesn't, technically that's script in groovy. But i will fix this, thanks

**Comment by rasom** [comment]
https://ci.status.im/job/status-react/job/desktop/job/pull-requests/job/PR-5479/2/

<blockquote><div><strong><a href="https://ci.status.im/job/status-react/job/desktop/job/pull-requests/job/PR-5479/2/">status-react » desktop » pull-requests » PR-5479 #2 [Jenkins]</a></strong></div></blockquote>

**Comment by lukaszfryc** [comment]
I quickly tested mobile apps and all looks fine there.

@churik please add "Tested - OK" once you finish testing desktop.

**Comment by churik** [comment]
@lukaszfryc 
installed Status-desktop, log in and checked `ubuntu-sever` process connections - no Mixpanel was found.

**Pull request merged**